### ISSUE TITLE
Unable to use JcePKCSPBEOutputEncryptorBuilder in bc fips 1.0.0

### ISSUE THREAD

**Issue by dstutz** [bcgit/bc-java#231]
I am getting an error attempting to use the above mentioned class.  I searched through the sample code for usage (bcfipsin100) and I'm getting the same error when I run Pem.java with bc-fips-1.0.0.jar and bcpkix-fips-1.0.0.jar on the classpath.

```
Exception in thread "main" org.bouncycastle.operator.OperatorCreationException: unable to create OutputEncryptor: null
	at org.bouncycastle.pkcs.jcajce.JcePKCSPBEOutputEncryptorBuilder.build(Unknown Source)
	at bcfipsin100.pbeks.Pem.writeEncryptedKey(Pem.java:86)
	at bcfipsin100.pbeks.Pem.main(Pem.java:149)
Caused by: java.lang.NullPointerException
	at org.bouncycastle.jcajce.provider.BaseCipher.engineGetParameters(Unknown Source)
	at javax.crypto.Cipher.checkCryptoPerm(Cipher.java:1020)
	at javax.crypto.Cipher.init(Cipher.java:1245)
	... 3 more

```
The following line will cause error:

```
new JcePKCSPBEOutputEncryptorBuilder(NISTObjectIdentifiers.id_aes256_CBC).setProvider("BCFIPS").build("keyPassword".toCharArray());
```

I've tried several different algorithms and it doesn't seem to matter.  I'm using JDK 1.8.0_144.

**Comment by bcgit** [comment]
This is issue BC-FJA-1.0.0-7 - it's fixed in 1.0.1, but you can work around it by making sure the unrestricted policy files are installed.

**Comment by dstutz** [comment]
Thank you! That did it.  (Yay for Java 9 coming with unrestricted out of the box!)
